resources/js/components/global/DashboardNotification.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="dashboard-notification padding-notification">
+    <div class="row">
+      <div class="dashboard-notification__message-container col-7">
+        <span class="dashboard-notification__message">Next module will be available on</span>
+        <span class="dashboard-notification__message-date">00/00/00</span>
+      </div>
+      <div class="dashboard-notification__schedule-button pointer col-5 align-self-end">
```

Comment:
You don't need the class of ".align-self-end" here, it's not doing anything

Reply:
thanks to the ``align-self-end`` we are able to put the schedule button at the bottom right of the element, which is more closely aligned to the mock-up. Without it, the button would go to the top-right